.github/workflows/cd-dev.yaml

```
@@ -0,0 +1,35 @@
+name: unionmate dev CD 파이프라인
+
+on:
+  workflow_run:
+    workflows: ["unionmate CI pipeline"]
+    types:
+      - completed
+
+jobs:
+  deploy:
+    if: >
```

actionlint (1.7.7)

if: condition "${{\n  github.event.workflow_run.conclusion == 'success' &&\n  github.event.workflow_run.event == 'push' &&\n  github.event.workflow_run.head_branch == 'dev' &&\n  github.event.workflow_run.head_repository.full_name == github.repository\n}}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)